src/diagnostics/riscv64/disasm-riscv64.cc

```diff
@@ -80,6 +80,10 @@ class Decoder {
   void PrintShamt(Instruction* instr);
   void PrintShamt32(Instruction* instr);
   void PrintRvcImm6(Instruction* instr);
+  void PrintRvcImm6Ldsp(Instruction* instr);
+  void PrintRvcImm6Lwsp(Instruction* instr);
+  void PrintRvcImm6Sdsp(Instruction* instr);
+  void PrintRvcImm6Swsp(Instruction* instr);
```

Comment:
If we decide to change the name of `instr->RvcImm6XXX()`, make the names consistent here